.travis.yml

```diff
@@ -41,6 +42,7 @@ install:
 
 script:
   - if [[ $TEST_COVERAGE ]]; then phpdbg -qrr bin/phpunit --coverage-clover clover.xml; else bin/phpunit; fi;
+  - if [[ $TEST_PHPSTAN ]]; then php ./bin/phpstan analyse -l 7 src fi;
```

Comment:
We should analyse `tests` directory too. I think it would be better to add the phpstan configuration and required extensions if necessary